medusa/reconstruct/expand.py

```diff
@@ -1,261 +1,72 @@
+#generating dictionary of the probablities for all csv files
 
-import itertools
-import random
-from itertools import chain
-
-from optlang.interface import OPTIMAL
-
-from cobra.flux_analysis.gapfilling import GapFiller
-from cobra.flux_analysis.parsimonious import add_pfba
-from cobra.core import DictList
-
-from cobra.util.solver import linear_reaction_coefficients
-
-from medusa.core.ensemble import Ensemble
-from medusa.core.feature import Feature
-from medusa.core.member import Member
-# functions for expanding existing models to generate an ensemble
-
-REACTION_ATTRIBUTES = ['lower_bound', 'upper_bound']
-MISSING_ATTRIBUTE_DEFAULT = {'lower_bound':0,'upper_bound':0}
+def probability_dictionary(file_directory):
+    """
+    This function takes the directory forprobability files 
+    and generate a reaction with their probabilities
+    """
+    reaction_probability = {}
+    probability_data = pd.read_csv(file_directory)
+    header_row = 0
+    probability_data = probability_data.transpose()
+    probability_data.columns = probability_data.iloc[header_row]
+    probability_data  = probability_data.drop('Unnamed: 0')
+    for w in probability_data.index:
+        f = float(probability_data.loc[w, "probability"])
+        reaction_probability[w] = f 
+    return reaction_probability
```

Comment:
This function is something that would be best for users to do with their own code rather than having a function provided by Medusa, since we don't know what program they'll be using to generate their own reaction probabilities. Rather than having a function like this, we should enforce usage of a dictionary of {reaction.id:probability} in any Medusa functions that use a probability dictionary.